packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -197,7 +199,19 @@ const tenantsRouter = (
     .post(
       "/tenants/:id",
       authorizationMiddleware([ADMIN_ROLE]),
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        try {
+          const id = await tenantService.updateTenant({
+            tenantId: req.params.id,
+            mailsSeed: req.body,
+            authData: req.ctx.authData,
+          });
+          return res.status(200).json({ id }).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, () => 500);
```

Comment:
shouldn't this return also a 403 and a 404?

Reply:
Fixed, thanks!